Sources/protoc-gen-swift/ExtensionGenerator.swift

```diff
@@ -170,11 +170,13 @@ struct ExtensionGenerator {
         p.print("set {setExtensionValue(ext: \(swiftFullExtensionName), value: newValue)}\n")
         p.outdent()
         p.print("}\n")
+        p.print("/// Returns true if extension `\(swiftFullExtensionName)`\n/// has been explicitly set.\n")
```

Comment:
In the regular fields, we currently don't; so I was copying that here.  My thought was just trying to keep things compact if someone does crack open the code to skim things.  I'm inclined to say we let it be in this form until we have a chance to get specific feedback from developers.